src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,37 @@
+import React, { useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo | null;
+  handlerModButton: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, handlerModButton }) => {
+  const [user, setUser] = React.useState<User | null>(null);
+  const [loading, setLoading] = React.useState(true);
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
The loading state is set to `true` twice: once during initialization and again at the start of the `useEffect`. The second setting is redundant and can be removed.